templates/index.liquid

```diff
@@ -6,9 +6,15 @@
   <div class="container">
     <div class="grid-row">
       {% for product in collection.products %}
-      <div class="grid-item item-s-12 item-m-4 item-l-3 text-align-center">
+      <div class="grid-item item-s-12 item-m-4 item-l-3 text-align-center margin-bottom-basic">
         <a href="{{ product.url }}" class="text-align-center">
-          <img src="{{ product.featured_image | product_img_url: 'x376' }}" alt="{{ product.title | escape }}" />
+          <img src="{{ product.featured_image | product_img_url: '300x', format: 'pjpg' }}" alt="{{ product.title | escape }}"
+          srcset="
+          {% if product.featured_image.width >= 300 %}{{ product.featured_image.src | product_img_url: '300x' }} 300w{% endif %}
+          {% if product.featured_image.width >= 720 %},{{ product.featured_image.src | product_img_url: '720x' }} 720w{% endif %}
```

Comment:
Because of the linebreak and space here the output markup looks a bit weird. Obviously it works fine. Not sure if we want to be too concerned about details like that?